client/consensus/common/src/lib.rs

```diff
@@ -18,7 +18,25 @@
 
 //! Collection of common consensus specific implementations
 
+#[macro_use]
+extern crate log;
```

Comment:
Is this really needed?

Reply:
log is used in these two files: https://github.com/gilescope/substrate/blob/2ba954e5c7db603ff395ca1d17bcabf61f7956da/client/consensus/common/src/import_queue/basic_queue.rs#L103 and https://github.com/gilescope/substrate/blob/2ba954e5c7db603ff395ca1d17bcabf61f7956da/client/consensus/common/src/import_queue.rs#L213

Reply:
Sure, but you don't need the obsolete `macro_use` and `extern crate` for this. Just `use log::debug;` or `log::debug!()`.

Reply:
updated imports.